Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/tests.model.ts

```diff
@@ -18,6 +18,7 @@ export class Test {
     public questionOrder: TestOrder;
     public optionOrder: TestOrder;
     categoryAcList: Category[] = [];
+    isQuestionMissisng: boolean;
```

Comment:
make sure your code should not affect other features of test creation. so do proper testing also.